stable/external-dns/values.yaml

```diff
@@ -282,10 +282,15 @@ rbac:
 ## Kubernetes Security Context
 ## https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
 ##
-securityContext:
-  enabled: true
+securityContext: {}
```

Comment:
Should we use a name such as `containerSecuirtyContext`? Just asking

Reply:
Security context for a Pod is backed by [PodSecurityContext](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#podsecuritycontext-v1-core) object and security context for a Container is backed by a [SecurityContext](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#securitycontext-v1-core) object. I would leave it this way.